### PR TITLE
Add service widget to firewall scenario

### PR DESCRIPTION
**Pull request by jknphy**
Adapt firewall scenario for integration of service configuration widget and remove soft-failures.

- Related ticket: https://progress.opensuse.org/issues/46262
- Verification run: [sle-15-SP1-yast2_gui](http://rivera-workstation.suse.cz/tests/1083)

### CODE REVIEW

**Comment by okurz** [comment]
I assume no needles are necessary because you just took over the needle reference from the template with "xyz". I have edited the description of the PR.

**Comment by okurz** [comment]
I can not access your verification run. Please check the referenced URL

**Comment by jknphy** [comment]
On Friday I was restarting openQA services on that machines. I guess you accessed just in that moment.

**Comment by okurz** [comment]
No, I mean "http://rivera-workstation/tests/1083" can not resolve to anything useful. But it's rivera-workstation.suse.cz . I think my links are all same kind of wrong assuming that my resolv.conf would automatically try all suse domains but it does not. If I may ask you then please provide full URLs, e.g. http://rivera-workstation.suse.cz/tests/1083

File: tests/yast2_gui/yast2_firewall.pm [hunk 1]
```diff
@@ -111,10 +93,7 @@ sub verify_service_started {
     select_console 'x11', await_console => 0;
     $self->launch_yast2_module_x11('firewall', target_match => 'firewall-start-page');
     assert_screen 'yast2_firewall_start-up';
-    assert_screen 'yast2_firewall_service_status_stopped';
-    send_key $fw{service_start};
-    detect_bsc_1114677 'yast2_firewall_service_status_running';
```
Comment (okurz) [inline tests/yast2_gui/yast2_firewall.pm]: the bug is still open, why remove the detection?

**Comment by jknphy** [comment]
Sometimes it helps to delete the resolve.conf (it happens to me that after manual edition it behaves strange). No problem, I will remember to be logged in with the fqdn.

**Comment by okurz** [comment]
LGTM

**Comment by nilxam** [comment]
not sure what's going on there https://openqa.opensuse.org/tests/834878#step/yast2_firewall/11 , please have a look.

**Comment by jknphy** [comment]
@nilxam sure, I will check it and label, there are some missing needles and probably another problem afterward like in https://openqa.suse.de/tests/2392579

**Pull request merged**